README.md

```diff
@@ -6,38 +6,39 @@ This is an open-ended pair lab! It is up to you and your teammate to come up wit
 
 ## Technical Requirements
 
-Your application should be ME(A)N. This includes...
+Your application should be MEAN. This includes...
 * Using **Express** to handle HTTP requests and serve API endpoints.
 * Using **MongoDB** as your database.
 * Using **Mongoose** as your ODM.
 * Adding **Angular** as your front-end. Express should only handle your root view as well as your API routes. Angular should take care of the rest via `ui.router`.
 
 Your app should have CRUD functionality for at least one model.
 
-
 ### Bonus I
 
 Add a second model to your application, which is nested under the first.
 
 ### Bonus II
 
-Allow the user to log-in using Twitter. [Lesson Plan](https://github.com/ga-wdi-lessons/express-oauth)
+Allow the user to log-in using Twitter.
+
+> Use this [lesson plan](https://github.com/ga-wdi-lessons/express-oauth) as reference.
 
 ## Pairs
 
-You will be working in pairs. We will leave the group selection up to you. Feel free find a partner, after the lab has been introduced.
+You will be working in pairs. We will leave the group selection up to you. Feel free find a partner after the lab has been introduced.
 
```

Comment:
Should we add any language here about how they should approach pair programming (I.E. a quick reminder of driver / navigator roles and whether or not they should switch off) or should we let them figure this out for themselves?